code/game/jobs/job/engineering.dm

```diff
@@ -43,6 +43,40 @@
 	dufflebag = /obj/item/weapon/storage/backpack/duffel/engineering
 	box = /obj/item/weapon/storage/box/engineer
 
+/datum/job/senioreng
+	title = "Senior Engineer" // Chartered Engineer?
+	flag = SENIORENG
+	department_flag = ENGSEC
+	total_positions = 1
+	spawn_positions = 1
+	is_engineering = 1
+	supervisors = "the chief engineer"
+	selection_color = "#fff5cc"
+	access = list(access_eva, access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels, access_external_airlocks, access_construction, access_atmospherics, access_mineral_storeroom)
+	minimal_access = list(access_eva, access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels, access_external_airlocks, access_construction, access_mineral_storeroom)
+	minimal_player_age = 30
+	exp_requirements = 3400 // This'll be changed to be karma related once I've worked that bit out.
+	exp_type = EXP_TYPE_CREW
+	outfit = /datum/outfit/job/senioreng
+
+/datum/outfit/job/senioreng // Since nothing actually changes, equipment-wise, could probably just use the regular datum

```

Comment:
You definitely should infact use the regular datum if nothing's changed and you don't plant to change anything.